gui/widgets/editable.cpp

```diff
@@ -340,7 +340,7 @@ void EditableWidget::drawCaret(bool erase) {
 		// possible glitches due to different methods used.
 		width = MIN(editRect.width() - caretOffset, width);
 		if (width > 0) {
-			g_gui.theme()->drawText(Common::Rect(x, y, x + width, y + editRect.height()), character,
+			g_gui.theme()->drawText(Common::Rect(x, y, x + width, y ), character,
```

Comment:
This change does not follow our Code Formatting conventions. Please, pay attention to the spaces around braces.